ditto/models/regulator.py

```diff
@@ -113,11 +113,10 @@ class Regulator(DiTToHasTraits):
     # These attributes are filled once the DiTTo model has been created using the Network module
     substation_name = Unicode(
         help="""The name of the substation to which the object is connected.""",
-        default=None,
-    )
+    ).tag(default=None)
     feeder_name = Unicode(
-        help="""The name of the feeder the object is on.""", default=None
-    )
+        help="""The name of the feeder the object is on.""",
+    ).tag(default=None)
```

Comment:
I think we should add a minimum bound in `setup.py` for `traitlets` for the version since when the old version was deprecated.